compiler/rustc_resolve/src/late.rs

```diff
@@ -239,14 +244,21 @@ enum LifetimeRibKind {
     /// `body_id` is an anonymous constant and `lifetime_ref` is non-static.
     AnonConst,
 
-    /// For **Modern** cases, create a new anonymous region parameter
-    /// and reference that.
+    /// Create a new anonymous region parameter and reference it.
```

Comment:
```suggestion
    /// Create a new anonymous lifetime parameter and reference it.
```
Could you normalize everything to the "lifetime" naming in name resolution?
That's what is already used here in majority of cases.